src/cmds/openapi.ts

```diff
@@ -227,12 +228,11 @@ export default class OpenAPICommand extends Command {
       Command.debug(`api settings list response payload: ${JSON.stringify(apiSettingsBody)}`);
       if (!apiSettingsBody.length) return createSpec();
 
-      const { option }: { option: 'create' | 'update' } = await prompt(
+      const { option }: Answers<string> = await promptTerminal(
         promptHandler.createOasPrompt(apiSettingsBody, parsedDocs, totalPages, getSpecs)
       );
-
       Command.debug(`selection result: ${option}`);
-      if (!option) return null;
```

Comment:
Is `option` never going to be `null` now?

Reply:
Yeah I couldn't figure out how to reproduce that when writing tests... if anything we should be throwing an error there rather than returning `null`. I imagine it's coming from our pagination stuff, which I added a little test TODO for here:
https://github.com/readmeio/rdme/blob/e5845e26a9f223c19f92eafe2299693d602589dc/__tests__/cmds/openapi.test.ts#L352